docs/src/neptune-core/rpc.md

```diff
@@ -0,0 +1,37 @@
+# RPC
+`neptune-core` provides an RPC server and client based on [tarpc](https://docs.rs/crate/tarpc) with [serde_json](https://docs.rs/crate/serde_json) transport. The RPC server provides a set of methods that allow clients to query the state of the node, submit transactions, and interact with the blockchain.
+
+As we use [tarpc](https://docs.rs/crate/tarpc/latest), it is easier to build a client in Rust. However, it is also possible to build a client in other languages that support JSON-RPC.
+
+The RPC server listens on the address specified by the `rpc-port` configuration option. The default address is `9799`.
+
+## Authentication 
+`neptune-core` currently supports cookie-based authentication. The RPC server provides a `cookie-hint` method that allows clients to locate the data-directory in a zero configuration way. If this method fails, the client should fall-back to the default data-directory location.
+
+## Versionning 
+Currently, we are on version `1.0` of the RPC server. This version is hardcoded in the RPC server and client. We will increment the version number if we make breaking changes to the RPC server.
```

Comment:
let's avoid use of the word "we".   just use a neutral tone such as Currently, neptune-core is at version... 

The version number will be be incremented...